new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/LoggingUtil.java

```diff
@@ -137,4 +232,162 @@ private static void writeToFile(Object obj, String resultDir, String fileName) t
     }
   }
 
-}
+  /**
+   * Copy files from persistent volume to local folder.
+   * @param namespace name of the namespace
+   * @param hostPath the persistent volume host path
+   * @param destinationPath destination folder to copy the files to
+   * @throws ApiException when pod interaction fails
+   */
+  private static void copyFromPV(String namespace, String hostPath, Path destinationPath) throws ApiException {
+    V1Pod pvPod = null;
+    try {
+      // create a temporary pod to get access to the interested persistent volume
+      pvPod = setupPVPod(namespace, hostPath);
```

Comment:
why do you need ns? why not just use default?

Reply:
I am creating the one-off pod in the same namespace the test used, also to create the one-off pod with a namespace appended to its name.

So even if there is a problem in one-off pod deletion, when the test runs the cleanup util everything in that test namespace gets wiped out.